tests/testAmpInfoTable.py

```diff
@@ -115,7 +115,7 @@ def testBasics(self):
             self.assertEquals(saturation, record.getSaturation())
             self.assertEquals(readNoise, record.getReadNoise())
             self.assertEquals(readoutCorner, record.getReadoutCorner())
-            self.assertEquals(linearityCoeffs, record.getLinearityCoeffs())
+            self.assertEquals(linearityCoeffs, tuple(record.getLinearityCoeffs()))
```

Comment:
I would recommend switching this to ```self.assertEquals(list(linearityCoeffs), record.getLinearityCoeffs())``` so that we are casting the type of the comparison tuple to the expected type of record.getLinearityCoeffs().